src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+.card {
+  padding: 33px 17px 17px;
+  border: 1px solid #f3f3f3;
+  width: 200px;
+  box-sizing: border-box;
+
+  transition: all 300ms;
+
+  &:hover {
+    transform: scale(1.2);
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), transformations should be smooth. Ensure that the scale transformation does not cause the card to overlap with adjacent elements, which can lead to a poor user experience.